nixos/modules/installer/tools/nix__-rebuild.sh

```diff
@@ -19,73 +31,80 @@ fast=
 rollback=
 upgrade=
 repair=
-profile=/nix/var/nix/profiles/system
 buildHost=
 targetHost=
+if [ "$flavour" = nixos ]; then
+    profile=/nix/var/nix/profiles/system
+else
+    profile=/nix/var/nix/profiles/nixup/$(whoami)/default
+fi
 
 while [ "$#" -gt 0 ]; do
     i="$1"; shift 1
-    case "$i" in
-      --help)
+    case "$i"_@flavour@ in
+      --help_*)
         showSyntax
         ;;
-      switch|boot|test|build|dry-build|dry-run|dry-activate|build-vm|build-vm-with-bootloader)
+      switch_*|boot_nixos|login_nixup|test_nixos|build_*|dry-build_nixos|dry-run_*|dry-activate_nixos|build-vm_nixos|build-vm-with-bootloader_nixos|edit_nixos)
```

Comment:
Doesn't this (and changes of commandline flags below) break the existing usage of `nixos-rebuild`?